### PR TITLE
Android: Use view binding

### PR DESCRIPTION
**Pull request by t895**
View binding is the successor to findViewById and ButterKnife injection. It is faster, reduces compile times, and has null/type safety. There aren't any visible changes in this PR besides some very slight text alignment changes in the convert activity.

More information about what view binding does and how to use it - https://developer.android.com/topic/libraries/view-binding#java

Note - These changes will not remove findViewById from the CustomTitleView for the TV layout nor the custom file picker due to limitations with leanback and nononsensefilepicker

### CODE REVIEW

**Comment by t895** [comment]
@dolphin-emu-bot rebuild

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/GameDetailsDialog.java [hunk 1]
```diff
@@ -34,31 +35,31 @@ public static GameDetailsDialog newInstance(String gamePath)
...
+    TextView textFileFormat = binding.textFileFormat;
+    TextView textCompression = binding.textCompression;
+    TextView textBlockSize = binding.textBlockSize;
 
-    TextView labelFileFormat = contents.findViewById(R.id.label_file_format);
-    TextView labelCompression = contents.findViewById(R.id.label_compression);
-    TextView labelBlockSize = contents.findViewById(R.id.label_block_size);
+    TextView labelFileFormat = binding.labelFileFormat;
+    TextView labelCompression = binding.labelCompression;
+    TextView labelBlockSize = binding.labelBlockSize;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/GameDetailsDialog.java:62]: I think it would be nicer to skip defining all these variables. Most of them only get used once or twice, and not having to write `binding.` isn't much of a gain.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatDetailsFragment.java [hunk 2]
```diff
@@ -42,31 +43,34 @@ public class CheatDetailsFragment extends Fragment
...
+    mEditCreatorLayout = mBinding.editCreator;
+    mEditCreator = mBinding.editCreatorInput;
+    mEditNotesLayout = mBinding.editNotes;
+    mEditNotes = mBinding.editNotesInput;
+    mEditCodeLayout = mBinding.editCode;
+    mEditCode = mBinding.editCodeInput;
+    mButtonDelete = mBinding.buttonDelete;
+    mButtonEdit = mBinding.buttonEdit;
+    mButtonCancel = mBinding.buttonCancel;
+    mButtonOk = mBinding.buttonOk;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatDetailsFragment.java:73]: Same here.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatListFragment.java [hunk 3]
```diff
@@ -16,24 +16,27 @@
...
 import com.google.android.material.divider.MaterialDividerItemDecoration;
 
-import org.dolphinemu.dolphinemu.R;
+import org.dolphinemu.dolphinemu.databinding.FragmentCheatListBinding;
 import org.dolphinemu.dolphinemu.features.cheats.model.CheatsViewModel;
 import org.dolphinemu.dolphinemu.utils.InsetsHelper;
 
 public class CheatListFragment extends Fragment
 {
+  private FragmentCheatListBinding mBinding;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatListFragment.java:25]: Is this needed?

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatViewHolder.java [hunk 4]
```diff
@@ -25,13 +25,13 @@ public class CheatViewHolder extends CheatItemViewHolder
...
   {
-    super(itemView);
+    super(binding.getRoot());
 
-    mRoot = itemView.findViewById(R.id.root);
-    mName = itemView.findViewById(R.id.text_name);
-    mCheckbox = itemView.findViewById(R.id.checkbox);
+    mRoot = binding.root;
+    mName = binding.textName;
+    mCheckbox = binding.checkbox;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatViewHolder.java:34]: These three variables could probably also be removed.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatsActivity.java [hunk 5]
```diff
@@ -86,13 +89,14 @@ protected void onCreate(Bundle savedInstanceState)
...
+    setContentView(mBinding.getRoot());
 
     WindowCompat.setDecorFitsSystemWindows(getWindow(), false);
 
-    mSlidingPaneLayout = findViewById(R.id.sliding_pane_layout);
-    mCheatList = findViewById(R.id.cheat_list);
-    mCheatDetails = findViewById(R.id.cheat_details);
+    mSlidingPaneLayout = mBinding.slidingPaneLayout;
+    mCheatList = mBinding.cheatList;
+    mCheatDetails = mBinding.cheatDetails;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatsActivity.java:99]: Maybe these ones too - you can decide

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/riivolution/ui/RiivolutionViewHolder.java [hunk 6]
```diff
@@ -26,13 +27,13 @@ public class RiivolutionViewHolder extends RecyclerView.ViewHolder
...
+  public RiivolutionViewHolder(@NonNull View itemView, ListItemRiivolutionBinding binding)
   {
     super(itemView);
 
-    mTextView = itemView.findViewById(R.id.text_name);
-    mChoiceLayout = itemView.findViewById(R.id.layout_choice);
-    mChoiceDropdown = itemView.findViewById(R.id.dropdown_choice);
+    mTextView = binding.textName;
+    mChoiceLayout = binding.layoutChoice;
+    mChoiceDropdown = binding.dropdownChoice;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/riivolution/ui/RiivolutionViewHolder.java:36]: These too (though you'll need an `mBinding` variable instead)

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java [hunk 7]
```diff
@@ -79,67 +84,68 @@ public SettingsAdapter(SettingsFragmentView view, Context context)
...
-        view = inflater.inflate(R.layout.list_item_setting, parent, false);
-        return new InputStringSettingViewHolder(view, this);
+        ListItemSettingBinding stringBinding = ListItemSettingBinding.inflate(inflater);
+        return new InputStringSettingViewHolder(stringBinding, this);
 
       case SettingsItem.TYPE_HYPERLINK_HEADER:
-        view = inflater.inflate(R.layout.list_item_header, parent, false);
-        return new HeaderHyperLinkViewHolder(view, this, mContext);
+        ListItemHeaderBinding hyperlinkBinding = ListItemHeaderBinding.inflate(inflater);
+        return new HeaderHyperLinkViewHolder(hyperlinkBinding, this);
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java:140]: Perhaps consider doing it like this instead? (Including the other occurrences above)

```suggestion
        return new HeaderHyperLinkViewHolder(ListItemHeaderBinding.inflate(inflater), this);
```

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/CheckBoxSettingViewHolder.java [hunk 8]
```diff
@@ -22,17 +22,21 @@ public final class CheckBoxSettingViewHolder extends SettingViewHolder
...
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
-    mCheckbox = root.findViewById(R.id.checkbox);
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
+    mCheckbox = mBinding.checkbox;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/CheckBoxSettingViewHolder.java:39]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/FilePickerViewHolder.java [hunk 9]
```diff
@@ -27,18 +28,21 @@ public final class FilePickerViewHolder extends SettingViewHolder
...
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
-
-    mDefaultBackground = root.getBackground();
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
+    mDefaultBackground = mBinding.getRoot().getBackground();
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/FilePickerViewHolder.java:45]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/HeaderViewHolder.java [hunk 10]
```diff
@@ -5,30 +5,35 @@
...
+    mBinding = binding;
+    findViews();
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mHeaderName = root.findViewById(R.id.text_header_name);
+    mHeaderName = mBinding.textHeaderName;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/HeaderViewHolder.java:32]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/InputBindingSettingViewHolder.java [hunk 11]
```diff
@@ -24,18 +25,22 @@ public final class InputBindingSettingViewHolder extends SettingViewHolder
...
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/InputBindingSettingViewHolder.java:43]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/InputStringSettingViewHolder.java [hunk 12]
```diff
@@ -20,16 +21,21 @@ public final class InputStringSettingViewHolder extends SettingViewHolder
...
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/InputStringSettingViewHolder.java:38]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/RumbleBindingViewHolder.java [hunk 13]
```diff
@@ -24,18 +25,22 @@ public class RumbleBindingViewHolder extends SettingViewHolder
...
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/RumbleBindingViewHolder.java:43]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/RunRunnableViewHolder.java [hunk 14]
```diff
@@ -25,18 +27,22 @@ public final class RunRunnableViewHolder extends SettingViewHolder
...
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/RunRunnableViewHolder.java:45]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SingleChoiceViewHolder.java [hunk 15]
```diff
@@ -23,16 +24,20 @@ public final class SingleChoiceViewHolder extends SettingViewHolder
...
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SingleChoiceViewHolder.java:40]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SliderViewHolder.java [hunk 16]
```diff
@@ -23,18 +25,22 @@ public final class SliderViewHolder extends SettingViewHolder
...
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
-    mTextSettingDescription = root.findViewById(R.id.text_setting_description);
+    mTextSettingName = mBinding.textSettingName;
+    mTextSettingDescription = mBinding.textSettingDescription;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SliderViewHolder.java:43]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SubmenuViewHolder.java [hunk 17]
```diff
@@ -18,15 +19,19 @@ public final class SubmenuViewHolder extends SettingViewHolder
...
+    mBinding = binding;
+    findViews();
   }
 
   @Override
-  protected void findViews(View root)
+  protected void findViews()
   {
-    mTextSettingName = root.findViewById(R.id.text_setting_name);
+    mTextSettingName = mBinding.textSettingName;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SubmenuViewHolder.java:34]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/sysupdate/ui/SystemUpdateProgressBarDialogFragment.java [hunk 18]
```diff
@@ -33,15 +33,16 @@ public Dialog onCreateDialog(@Nullable Bundle savedInstanceState)
     SystemUpdateViewModel viewModel =
             new ViewModelProvider(requireActivity()).get(SystemUpdateViewModel.class);
 
-    View dialogView = getLayoutInflater().inflate(R.layout.dialog_progress, null, false);
-    LinearProgressIndicator progressBar = dialogView.findViewById(R.id.update_progress);
+    DialogProgressBinding dialogProgressBinding =
+            DialogProgressBinding.inflate(getLayoutInflater());
+    LinearProgressIndicator progressBar = dialogProgressBinding.updateProgress;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/sysupdate/ui/SystemUpdateProgressBarDialogFragment.java:38]: `progressBar` variable could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/ConvertFragment.java [hunk 19]
```diff
@@ -195,15 +199,14 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState)
...
-    mCompressionLevelDropdown = requireView().findViewById(R.id.dropdown_compression_level);
-    mRemoveJunkDataCheckbox = requireView().findViewById(R.id.checkbox_remove_junk_data);
+    mFormatDropdown = mBinding.dropdownFormat;
+    mBlockSizeLayout = mBinding.blockSize;
+    mBlockSizeDropdown = mBinding.dropdownBlockSize;
+    mCompressionLayout = mBinding.compression;
+    mCompressionDropdown = mBinding.dropdownCompression;
+    mCompressionLevelLayout = mBinding.compressionLevel;
+    mCompressionLevelDropdown = mBinding.dropdownCompressionLevel;
+    mRemoveJunkDataCheckbox = mBinding.checkboxRemoveJunkData;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/ConvertFragment.java:209]: Could be removed. I do think these are used more often than the stuff I've commented on in previous files, but still, it's just a matter of typing `mBinding.`.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/ConvertFragment.java [hunk 20]
```diff
@@ -466,15 +476,16 @@ private void convert(String outPath)
 
     mCanceled = false;
 
-    View dialogView = getLayoutInflater().inflate(R.layout.dialog_progress, null, false);
-    LinearProgressIndicator progressBar = dialogView.findViewById(R.id.update_progress);
+    DialogProgressBinding dialogProgressBinding =
+            DialogProgressBinding.inflate(getLayoutInflater(), null, false);
+    LinearProgressIndicator progressBar = dialogProgressBinding.updateProgress;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/ConvertFragment.java:481]: `progressBar` variable could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java [hunk 21]
```diff
@@ -79,23 +84,26 @@ private int getBottomPaddingRequired()
...
+    return mBinding.getRoot();
+  }
 
-    mPauseEmulation = options.findViewById(R.id.menu_pause_emulation);
-    mUnpauseEmulation = options.findViewById(R.id.menu_unpause_emulation);
+  @Override
+  public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState)
+  {
+    mPauseEmulation = mBinding.menuPauseEmulation;
+    mUnpauseEmulation = mBinding.menuUnpauseEmulation;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java:95]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainActivity.java [hunk 22]
```diff
@@ -154,14 +158,13 @@ else if (DirectoryInitialization.areDolphinDirectoriesReady())
...
-    mAppBarLayout = findViewById(R.id.appbar_main);
-    mToolbar = findViewById(R.id.toolbar_main);
-    mViewPager = findViewById(R.id.pager_platforms);
-    mTabLayout = findViewById(R.id.tabs_platforms);
-    mFab = findViewById(R.id.button_add_directory);
+    mAppBarLayout = mBinding.appbarMain;
+    mToolbar = mBinding.toolbarMain;
+    mViewPager = mBinding.pagerPlatforms;
+    mTabLayout = mBinding.tabsPlatforms;
+    mFab = mBinding.buttonAddDirectory;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainActivity.java:167]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/platform/PlatformGamesFragment.java [hunk 23]
```diff
@@ -123,9 +134,9 @@ public void setRefreshing(boolean refreshing)
     mSwipeRefresh.setRefreshing(refreshing);
   }
 
-  private void findViews(View root)
+  private void findViews()
   {
-    mSwipeRefresh = root.findViewById(R.id.swipe_refresh);
-    mRecyclerView = root.findViewById(R.id.grid_games);
+    mSwipeRefresh = mBinding.swipeRefresh;
+    mRecyclerView = mBinding.gridGames;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/platform/PlatformGamesFragment.java:140]: Could be removed

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/viewholders/GameViewHolder.java [hunk 24]
```diff
@@ -24,15 +20,15 @@ public class GameViewHolder extends RecyclerView.ViewHolder
...
+    binding.getRoot().setTag(this);
 
-    imageScreenshot = itemView.findViewById(R.id.image_game_screen);
-    textGameTitle = itemView.findViewById(R.id.text_game_title);
-    textGameTitleInner = itemView.findViewById(R.id.text_game_title_inner);
-    textGameCaption = itemView.findViewById(R.id.text_game_caption);
+    imageScreenshot = binding.imageGameScreen;
+    textGameTitle = binding.textGameTitle;
+    textGameTitleInner = binding.textGameTitleInner;
+    textGameCaption = binding.textGameCaption;
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/viewholders/GameViewHolder.java:32]: Could be removed.

Though, hmm... Does this class have any purpose left after removing these variables?

File: Source/Android/app/src/main/res/layout-ldrtl/list_item_cheat.xml [hunk 25]
```diff
@@ -1,37 +0,0 @@
...
-        android:layout_height="64dp"
-        android:focusable="true"
-        android:gravity="center"
-        android:nextFocusRight="@id/root"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toEndOf="@id/text_name"
-        app:layout_constraintTop_toTopOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (JosJuice) [inline Source/Android/app/src/main/res/layout-ldrtl/list_item_cheat.xml:37]: Why was this file deleted?

Note: This file swaps nextFocusLeft and nextFocusRight compared to the LTR file. If that's non-obvious, perhaps it would be nice to add a comment about it in the file?

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatListFragment.java (hunk 3 above)
Comment (t895) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatListFragment.java:25]: Yes, so we can set the binding to null when the fragment is destroyed.

File: Source/Android/app/src/main/res/layout-ldrtl/list_item_cheat.xml (hunk 25 above)
Comment (t895) [reply Source/Android/app/src/main/res/layout-ldrtl/list_item_cheat.xml:37]: Ah I didn't notice it before. I added a comment.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatListFragment.java (hunk 3 above)
Comment (JosJuice) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatListFragment.java:24]: I think I missed the reference to `mBinding.cheatList`.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SettingViewHolder.java [hunk 26]
```diff
@@ -57,11 +55,10 @@ protected static void showNotRuntimeEditableError()
...
 
   /**
-   * Gets handles to all this ViewHolder's child views using their XML-defined identifiers.
-   *
-   * @param root The newly inflated top-level view.
+   * Gets handles to all this ViewHolder's child views using their view bindings. Must be called
+   * in the constructor of the class that implements a SettingViewHolder.
    */
-  protected abstract void findViews(View root);
+//  protected abstract void findViews();
```
Comment (JosJuice) [inline Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SettingViewHolder.java:61]: Please remove this instead of just commenting it out.

**Review by JosJuice — APPROVED** [review]
LGTM other than that one comment.

File: Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SettingViewHolder.java (hunk 26 above)
Comment (t895) [reply Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/SettingViewHolder.java]: My bad
